### PR TITLE
3926: Back button return url included

### PR DESCRIPTION
**Pull request by RuthBerry96**

### CODE REVIEW

File: src/NHSD.BuyingCatalogue.Identity.Api/Views/Account/ForgotPassword.cshtml [hunk 1]
```diff
@@ -10,6 +10,14 @@
         <div data-test-id="login-page">
             <div class="nhsuk-grid-row">
                 <div class="nhsuk-grid-column-two-thirds">
+                    <div data-test-id="go-back-section" class="nhsuk-back-link">
```
Comment (kat1906) [inline src/NHSD.BuyingCatalogue.Identity.Api/Views/Account/ForgotPassword.cshtml:13]: you don't need `data-test-id` on every div - we just have the one below

Comment (kat1906) [reply src/NHSD.BuyingCatalogue.Identity.Api/Views/Account/ForgotPassword.cshtml:13]: (unless you are using it in your tests)

Comment (RuthBerry96) [reply src/NHSD.BuyingCatalogue.Identity.Api/Views/Account/ForgotPassword.cshtml:13]: It's being used in the tests

File: tests/NHSD.BuyingCatalogue.Identity.Api.IntegrationTests/Features/Forgot Password UI Validation.3926.feature [hunk 2]
```diff
@@ -57,4 +57,4 @@ Scenario: 7. The NHS Footer is displayed
 @3926
 Scenario: 8. The NHS Legal is displayed
 	Then the page contains element with Data ID nhsuk-legal-panel
-	And element with Data ID nhsuk-legal-panel contains a link to https://digital.nhs.uk/about-nhs-digital/privacy-and-cookies
\ No newline at end of file
+	And element with Data ID nhsuk-legal-panel contains a link to https://digital.nhs.uk/about-nhs-digital/privacy-and-cookies
```
Comment (bjss-chris) [inline tests/NHSD.BuyingCatalogue.Identity.Api.IntegrationTests/Features/Forgot Password UI Validation.3926.feature:60]: 👍

The file just needs untabifying now.

**Pull request merged**